src/SamplesApp/SamplesApp.iOS/SamplesApp.iOS.csproj

```diff
@@ -107,9 +107,7 @@
     <None Include="Info.plist" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Content Include="Entitlements.plist" />
-    <BundleResource Include="Resources\Fonts\winjs-symbols.ttf">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
I think you should leave this line there, unless someone can confirm that it's not necessary.

Reply:
I am removing all dependency on winjs-symbols. So why would we still need this?

Reply:
what I mean is the `<CopyToOutputDirectory>Always</CopyToOutputDirectory>` line. I don't know what it does - but that's why I wouldn't remove it.

Reply:
It's actually required for the bundling of fonts to work (or at least it used to).